proto/proxy/telemetry/telemetry.proto

```diff
@@ -67,11 +67,11 @@ message RequestCtx {
   string path = 5;
 }
 
-// A latency value in tenths of a millisecond and a count of the times
-// that latency was observed.
-message Latency {
-    // Latency value in tenths of a millisecond.
-    uint32 latency = 1;
+// A latency bucket, with a maximum value in tenths of a millisecond and a
+// count of the times that latency was observed.
+message LatencyBucket {
+    // Maximum latency value in this bucket, value in tenths of a millisecond.
+    uint32 max_value = 1;
```

Comment:
Since we're using fixed bucket histograms, we don't actually need to specify the bounds on every single histogram since they'll all be the same.  This lets us dramatically reduce the size of each histogram.